.github/workflows/job-postcommit-java-validatesrunner-flink-java11.yml

```diff
@@ -0,0 +1,87 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+name: Java VR Flink Java 11 PostCommit
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: ['v*']
+permissions: read-all
+
+jobs:
+  set-properties:
+    runs-on: [self-hosted, ubuntu-20.04]
+    outputs:
+      properties: ${{ steps.test-properties.outputs.properties }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - id: test-properties
+        uses: ./.github/actions/setup-default-test-properties
+
+  java_vr_flink_java11:
+    needs: set-properties
+    name: Run Flink ValidatesRunner Java 11
+    runs-on: [self-hosted, ubuntu-20.04]
+    strategy:
+      fail-fast: false
+      matrix:
+        flink-versions: ${{fromJson(needs.set-properties.outputs.properties).JavaTestProperties.FLINK_VERSIONS}}
+        exclude: # exclude other versions, we only need 1.13 for this test
```

Comment:
If we only need 1.13 should we just not use the matrix and use a variable instead?

Reply:
Same question for the other workflows

Reply:
It is just for when someone adds more versions for testing Flink and gets from one file the versions to run... @damccorm

Reply:
@damccorm do you have any comment on this?

Reply:
> we only need 1.13 for this test

Could you elaborate on this? Why do we only need 1.13? If its because we're not really testing flink's full matrix then we probably wouldn't want to pick up the new versions either

Reply:
yes, makes sense, in the original job it only picked up the `1.13 flink version`, so I would change the workflow to only run with that specific version

Reply:
Fix applied @damccorm